api/src/data_migration/load/load_oracle_data_task.py

```diff
@@ -253,8 +271,14 @@ def main() -> None:
         foreign_tables = {t.name: t for t in src.db.models.foreign.metadata.tables.values()}
         staging_tables = {t.name: t for t in src.db.models.staging.metadata.tables.values()}
 
+        columns_to_exclude = {
+            "tuser_account_mapper": ["email_address"],
+        }
```

Comment:
We want to exclude the email address from the `tuser_account` table, I mixed up the table names previously in that thread.

I also think we need to figure out where we want this configuration - if it's not a parameter we can override on the CLI (is there ever a case we'd want to do that?) then I'm fine just defining it statically in the LoadOracleDataTask's init function.